### ISSUE TITLE
Install preview won't do session logout

### ISSUE THREAD

**Issue by philipp1184** [networktocode/fortimanager-ansible#52]
i have recently create a playbook with 2 tasks. the first task is making a preview and the second will install it. the second tasks waiting for a prior session to finish.

**Comment by jmcgill298** [comment]
@philipp1184 is your issue that your second task never kicks off? Does the first task fail? Can you provide the playbook and any error messages.

**Comment by philipp1184** [comment]
Yes the second tasks does not start, because its blocked by the previous session

[install_w_preview.yml.txt](https://github.com/networktocode/fortimanager-ansible/files/1604250/install_w_preview.yml.txt)

i found out this is because in fortimgr_install.py the logout handling is only made if not preview. i changed the lines for me. see the patch file
[fortimgr_install.patch.txt](https://github.com/networktocode/fortimanager-ansible/files/1604258/fortimgr_install.patch.txt)

**Comment by jmcgill298** [comment]
That looks like the correct placement of the logout, but not sure why that would prevent the second task from executing. The logout functionality was an enhancement I made after the modules were all working to limit the session count on the device, but not necessary for multiple task executions.

Please submit a PR for the enhancement.

**Comment by jmcgill298** [comment]
@philipp1184 do you want to put a PR in for that patch?

**Comment by jmcgill298** [comment]
I have submitted/merged a PR to fix this
